api/mapping/featuredata/event/WFSSetFilter.md

```diff
@@ -1,2 +1,3 @@
 # WFSSetFilter
 
+Set WFS filter
```

Comment:
I think we should explain in the one-liner what the event does instead of just typing out the name. For example here: "Notifies that a GeoJSON geometry filter has been set"